eng/ci/templates/official/jobs/run-benchmarks.yml

```diff
@@ -128,3 +158,4 @@ jobs:
       crank compare "$(baselineBenchmarkResultFilePath)" "$(benchmarkResultsJsonPath)"
     condition: and(succeeded(), ne(variables['baselineBenchmarkResultFilePath'], ''))
     displayName: 'Compare Results with Baseline'
+    
```

Comment:
is there whitespace here or something? Wonder why it is giving the incorrect end-of-file icon.

Reply:
there was whitespace, which I removed in latest iteration.